lldb/source/Target/RegisterContextUnwind.cpp

```diff
@@ -2088,6 +2088,13 @@ bool RegisterContextUnwind::ReadFrameAddress(
     UnwindLogMsg("No suitable CFA found");
     break;
   }
+  case UnwindPlan::Row::FAValue::isConstant: {
+    address = fa.GetConstant();
+    if (abi_sp)
+      address = abi_sp->FixCodeAddress(address);
```

Comment:
We prefer people call `Process::FixDataAddress()` these days, a method which gets the ABI and runs the method.  (this is the address on the stack, so FixDataAddress is the correct method).  Yeah the code above at line 2051 should be doing the same thing, I know you were only copying it here.

Reply:
Sounds good! I will push an updated version calling the process method!